cli/Makefile

```
@@ -144,8 +148,14 @@ $(build_bindir)/julia$(EXE): $(EXE_OBJS) $(build_shlibdir)/libjulia.$(SHLIB_EXT)
 $(build_bindir)/julia-debug$(EXE): $(EXE_DOBJS) $(build_shlibdir)/libjulia-debug.$(SHLIB_EXT) | $(build_bindir)
 	@$(call PRINT_LINK, $(CC) $(LOADER_CFLAGS) $(DEBUGFLAGS) $(EXE_DOBJS) -o $@ $(LOADER_LDFLAGS) $(RPATH) -ljulia-debug)
 
+$(BUILDDIR)/julia.generated.expmap:
+	@echo "$(DEFAULT_SYMBOL_VERSION)" > $@.tmp
+	@echo "{ global: *; };" >> $@.tmp
```

This appears to disregard the content of the existing version-script, which should be preserved
```suggestion
$(BUILDDIR)/julia.generated.expmap: $(SRCDIR)/julia.expmap
	@echo "$(DEFAULT_SYMBOL_VERSION)" > $@.tmp
	@cat "$<" >> $@.tmp
```

---

Ah, you'd like to re-use the `src/julia.expmap` as a template for the CLI loader version script?

Sounds good to me, I'll make that change

---

Actually, I misunderstood this request.

@staticfloat would you like me to add a `julia.expmap` to the `cli/` directory or to re-use the existing one in `src/`?

If the latter, what's the safe way to reach across directories in our Makefiles?

---

We attach the symbol versioning in both Makefiles - the difference is that `cli/` before this change didn't use any version script, so it was globally exporting all of its symbols by default.

It still does that with this change, but now it uses a linker script with `{ global: *; };`